recipes/minimac4/build.sh

```
@@ -0,0 +1,41 @@
+#!/bin/bash
+#set -euo pipefail
+
+echo "Building Minimac4 version ${PKG_VERSION}"
+
+# Set up dependencies and directories
+cget ignore xz
+cget install -f ./requirements.txt
+cmake --version
+mkdir -p build
+cd build
```

Shellcheck

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)